specification/search/data-plane/Microsoft.Azure.Search.Service/track1/preview/2019-05-06-preview/searchservice.json

```diff
@@ -4272,6 +4277,33 @@
         }
       }
     },
+    "IndexerCache" : {
+      "properties" : {
+        "ID" : {
+          "type" : "string",
+          "readOnly": true,
+          "x-nullable": true,
+          "description" : "Server-assigned guid identifier for this cache."
+        },
+        "enableReprocessing" : {
+          "type" : "boolean",
+          "x-nullable": true,
+          "default": true,
+          "description" : "Flag asto wether skillset updates should reproecss old documents automatically."
+        },
+        "storageConnectionString": {
+          "type": "string",
+          "description": "The connection string to the blob storage account that will be used to store the cache information."
+        }
+      },
+      "required": [
+        "storageConnectionString"
+      ],
+      "description": "Represents a annotation cache used to store intermediate skill outputs to allow for incremental enrichment.",
```

Comment:
Wording here is different from what was used on the cache description. Are we publicly calling it "annotation cache" or "indexer cache"?

Reply:
indexer cache

---
In reply to: [348707934](https://github.com/Azure/azure-rest-api-specs/pull/7812#discussion_r348707934) [](ancestors = 348707934)